spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -14,7 +14,7 @@
 from qtpy import PYSIDE2
 from qtpy.QtCore import Signal, Qt
 from qtpy.QtGui import QFontMetricsF
-from qtpy.QtWidgets import QInputDialog, QLabel
+from qtpy.QtWidgets import QInputDialog, QLabel, QAction, QLineEdit
```

Comment:
```suggestion
from qtpy.QtWidgets import QAction, QInputDialog, QLabel, QLineEdit
```

Sort alphabetically.